dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -633,6 +633,38 @@ open class Jpql : JpqlDsl {
         return Expressions.index(entity.toEntity())
     }
 
+    /**
+     * Creates an expression that represents the id of the entity.
+     */
+    @SinceJdsl("3.6.0")
+    fun <ID : Any> id(entity: KClass<*>): Expression<ID> {
+        return Expressions.id(Entities.entity(entity))
+    }
+
+    /**
+     * Creates an expression that represents the id of the entity.
+     */
+    @SinceJdsl("3.6.0")
+    fun <ID : Any> id(entity: Expressionable<*>): Expression<ID> {
+        return Expressions.id(entity)
+    }
+
+    /**
+     * Creates an expression that represents the version of the entity.
+     */
+    @SinceJdsl("3.6.0")
+    fun <V : Any> version(entity: KClass<*>): Expression<V> {
+        return Expressions.version(Entities.entity(entity))
+    }
```

Comment:
Since id uses the name of the Generic type as ID, I think it would be okay to use the name of the Generic type as VERSION for version as well.

Reply:
yeah. applied.